server/httpin/restapi/restapi.go

```diff
@@ -36,6 +36,7 @@ func Init(router *mux.Router, conf config.Service, proxy proxy.Service, appServi
 		httputils.CheckAuthorized(mm, a.handleCall)).Methods("POST")
 
 	subrouter.HandleFunc(mmclient.PathSubscribe, a.handleSubscribe).Methods("POST")
+	subrouter.HandleFunc(mmclient.PathSubscribe, a.handleGetSubscriptions).Methods("GET")
```

Comment:
Is there a corresponding method in `ClientPP`?

Reply:
@hanzei This was mainly for debugging, but it should be put there you're right. I'll follow up with this change.

FYI we will likely be reworking how subscribing works, see https://community-daily.mattermost.com/core/pl/wu83zyfgdidgxbsdc9cftpyuto and https://mattermost.atlassian.net/browse/MM-37929